src/util/spreadsheet/misc.py

```diff
@@ -0,0 +1,21 @@
+import json
+
+
+def iter_commands(cr, revision_ids):
```

Comment:
From the current scripts that will use this function, we could change the signature to take the search patterns instead of the revision ids.

However, we will need to handle the `and` and `or` operators.
Proposal:
```suggestion
def iter_commands(cr, like_all=(), like_any=()):
    if like_all and like_any:
        raise ValueError("Please specify `like_all` or `like_any`, not both")
    where = "true"
    if like_all:
        where = cr.mogrify("commands LIKE ALL(%s)", [list(like_all)])
    elif like_any:
        where = cr.mogrify("commands LIKE ANY(%s)", [list(like_any)])
        
    cr.execute("SELECT id, commands FROM spreadsheet_revision WHERE " + where)
    for revision_id, data in cr.fetchall():
        ...
```